From 5228f94378291f2da780317bdae02bbe90be9104 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:36:26 -0400
Subject: [block] blk-mq: blk_mq_unregister_hctx() can be static

Message-id: <1402670238-13916-99-git-send-email-snitzer@redhat.com>
Patchwork-id: 83838
O-Subject: [RHEL7.1 PATCH 098/150] blk-mq: blk_mq_unregister_hctx() can be static
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit ee3c5db0896d85187b5f31b5482ed8fd308d31ee
Author: Fengguang Wu <fengguang.wu@intel.com>
Date:   Fri May 30 10:31:13 2014 -0600

    blk-mq: blk_mq_unregister_hctx() can be static

    CC: Jens Axboe <axboe@kernel.dk>
    Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
    Signed-off-by: Jens Axboe <axboe@fb.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-mq-sysfs.c b/block/blk-mq-sysfs.c
index e5f575f..ed52178 100644
--- a/block/blk-mq-sysfs.c
+++ b/block/blk-mq-sysfs.c
@@ -327,7 +327,7 @@ static struct kobj_type blk_mq_hw_ktype = {
  .release = blk_mq_sysfs_release,
 };
 
-void blk_mq_unregister_hctx(struct blk_mq_hw_ctx *hctx)
+static void blk_mq_unregister_hctx(struct blk_mq_hw_ctx *hctx)
 {
  struct blk_mq_ctx *ctx;
  int i;
@@ -341,7 +341,7 @@ void blk_mq_unregister_hctx(struct blk_mq_hw_ctx *hctx)
  kobject_del(&hctx->kobj);
 }
 
-int blk_mq_register_hctx(struct blk_mq_hw_ctx *hctx)
+static int blk_mq_register_hctx(struct blk_mq_hw_ctx *hctx)
 {
  struct request_queue *q = hctx->queue;
  struct blk_mq_ctx *ctx;
-- 
1.7.1